documentation/technical_writing_slides.md

```diff
@@ -0,0 +1,452 @@
+they won’t use it.
+
+Taken from [Write The Docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/#why-write-docs)
+
+---
+
+Similar message and highly recommended:
+Bangerth and Heister, 2013: [What makes computational open source software libraries successful?](http://dx.doi.org/10.1088/1749-4699/6/1/015010)
+
+---
+
+## Learning Goals
+
+- Raise **awareness** that documentation is crucial in RSE.
+- Understand that there is a difference between documentation and **good documentation**.
+- Get to know the purpose and basic structure of several **standard documentation building blocks** (README, commit message, changelog, ...).
+- **Not**: Learn how to really write good documentation. This takes practice and much more than 90 minutes.
+- **Not**: Any technical tools. This will come next week.
+
+---
+
+## Technical Writer
+
+- ... is a job profile between technical know-how (computer science, engineering, ...) and language.
+- ... can be a role in a software development team (*"documentation"*).
+- There is whole community, e.g. [Write the Docs](https://www.writethedocs.org/).
+
+Much content of this lecture is taken from [Write the Docs](https://www.writethedocs.org/).
+
+---
+
+## Technical Writing in Research Software
+
+- Like in a startup, **no dedicated technical writer in team** (even for large projects). Everybody does everything.
+    - -> You need writing skills.
+- In research, we anyway write a lot (papers, proposals, lecture material, ...).
+    - -> Writing skills are very important anyway.
+
+---
+
+## Why Write Documentation
+
+- You will be using your code in 6 months.
+- You want people to use your code.
+- You want people to help out.
+- You want your code to be better (act of putting words to paper requires a distillation of thought that may not be so easy).
+- You want to be a better writer.
+
+---
+
+If someone starts using your code ...
+
+## *I made something of value!*
+
+---
+
+## *What if it breaks?!*
+
+---
+
+## *I am a real open source developer!*
+
+---
+
+## *Oh god, someone else is using my code...*
+
+So, better write good documentation.
+
+---
+
+## 2. What is Good Documentation?
+
+---
+
+## Documentation Content Should Be
+
+- **ARID** (*"Accept (some) Repetition in Documentation"*)
+    - Repetition from code to docs
+    - Not everything can be auto-generated.
+- **Skimmable**
+    - Headings: should be descriptive and concise
+    - Hyperlinks: descriptive, not [click here](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/)
+    - Use listings etc. where appropriate, no *"wall of text"*
+- **Exemplary** (some, but not too many)
+- **Consistent** (language, formatting, wording)
+- **Current**
+
+---
+
+## Be Clear About Your Audience (1/2)
+
+**User vs. developer documentation**:
+
+- Distinction important for larger projects, not so important for smaller projects
+- Sometimes also third category *"maintainer"*
+- User docs: How does the software work?
```

Comment:
This is a bit ambiguous (to me). To me it sounds at least as if the user is interested in the internals of the software.
Why is it not: "What does the software do?" and "How to use the software?"